code/mob/living/intangible/ai-camera.dm

```diff
@@ -127,6 +118,30 @@
 		if(src.loc.z != 1)	//you may only move on the station z level!!!
 			src.cancel_camera()
 
+	proc/add_all_statics()
+		if (!src.loc)
+			return
+		for (var/turf/T as anything in block(src.loc.x - 11, src.loc.y - 8, src.loc.z, src.loc.x + 11, src.loc.y + 8, src.loc.z))
```

Comment:
These should get values from `client.view` instead of being magic numbers. In testing it seems like the edges of the screen flicker a bit so maybe add +1 all around to extend just past the edge of the screen.